templates/zerver/help/see-if-user-here-away.md

```diff
@@ -0,0 +1,35 @@
+### User List
+
+You can see if a user is active on Zulip by looking at the user list, located on the far right side of your screen.
+
+![User List](/static/images/help/user-list.png)
+
+The **Search people** field at the top of the user list allows you to search for a specific user. As you type the user's name, the user list will shrink, to show only people with that name.
+
+![Search Field](/static/images/help/search-field-user-list.png)
+
+If you mouse over a user's name on the user list, you will see a small chevron(![chevron](/static/images/help/chevron.png)).
+
+Clicking on this chevron brings up a menu like the one below.
+
+![Name Menu](/static/images/help/menu-name-in-user-list.png)
+
+User status is marked by little circles to the left of a user’s name:
+
+ * A green circle
+   (<span class="indicator green solid"></span>) means the
```

Comment:
There's some images of the circles in the `/static/images/help` folder, including
green_circle ![green_circle](https://cloud.githubusercontent.com/assets/15116870/21377508/02c2950c-c6f4-11e6-8a45-58f9ab3c3995.png), orange_half_circle ![orange_half_circle](https://cloud.githubusercontent.com/assets/15116870/21377513/052ceb58-c6f4-11e6-8d69-ae82f5d86971.png), pale_green_circle ![pale_green_circle](https://cloud.githubusercontent.com/assets/15116870/21377514/0792327c-c6f4-11e6-9f4b-e39af7a57215.png), and white_empty_circle ![white_empty_circle](https://cloud.githubusercontent.com/assets/15116870/21377518/0a7e3530-c6f4-11e6-8f7b-e0861682bbb2.png) (all .png extensions)